.github/workflows/doc.yml

```
@@ -53,6 +53,28 @@ jobs:
     - name: 'Build HTML documentation'
       run: make -C Doc/ SPHINXOPTS="-q" SPHINXERRORHANDLING="-W --keep-going" html
 
+    # Add pull request annotations for Sphinx nitpicks (missing references)
+    - name: Get list of changed files
+      id: changed_files
+      uses: Ana06/get-changed-files@v2.2.0
+    - name: 'Build changed files in nit-picky mode'
+      continue-on-error: true
+      run: |
+        # Mark files the pull request modified
+        touch ${{ steps.changed_files.outputs.added_modified }}
+        # Build docs with the '-n' (nit-picky) option; convert warnings to annotations
+        make -C Doc/ PYTHON=../python SPHINXOPTS="-q -n --keep-going" html 2>&1 |
+            python Doc/tools/warnings-to-gh-actions.py
+
+    # Ensure some files always pass Sphinx nit-picky mode (no missing references)
+    - name: 'Build known-good files in nit-picky mode'
```

Minor nit: does this need quotes?
```suggestion
    - name: Build known-good files in nit-picky mode
```

---

I don't _think_ it needs them (but with YAML's arcane coercion rules, who knows?) but the convention seems to be to use quotes on all the other `name` values (except for one of them added here, ironically), perhaps because they are human-readable UI text rather than something with programmatic meaning.